src/service/gateway.rs

```diff
@@ -65,10 +65,10 @@ impl Response for GatewayRespV1 {
     fn region_params(&self) -> Result<RegionParams> {
         match &self.msg {
             Some(gateway_resp_v1::Msg::RegionParamsStreamedResp(params)) => {
-                RegionParams::try_from(params.to_owned())
+                Ok(RegionParams::try_from(params.to_owned())?)
```

Comment:
surprised clippy doesn't have a problem with this; does the unwrap contain and convert the error case to the context of this module so it's a different error when it reaches the caller that if you just return the raw result of the `try_from`?

Reply:
right.. `RegionParams::try_from` returns a `beacon::Error` which doesn't auto convert to a gateway-rs Error. 

Note though this is _not_ an unwrap..